{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='grafana-0.grafana-endpoints.cos-lite.svc.cluster.local', port=3000): Max retries exceeded with url: /api/search?query=5G%20Network%20Overview (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x775f04c909d0>: Failed to resolve 'grafana-0.grafana-endpoints.cos-lite.svc.cluster.local' ([Errno -3] Temporary failure in name resolution)\"))","children":[{"name":"test_given_cos_lite_integrated_with_sdcore_when_searching_for_5g_network_overview_dashboard_in_grafana_then_dashboard_exists","uid":"a7416f36f73f225c","parentUid":"d46937ae06ef1fc18eceed9e2f45fe93","status":"broken","time":{"start":1741171691817,"stop":1741171694790,"duration":2973},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":["asyncio","abort_on_fail"]}],"uid":"d46937ae06ef1fc18eceed9e2f45fe93"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}